src/internal/remote/toBKTException.ts

```diff
@@ -15,30 +15,35 @@ import { FetchResponseLike } from './fetch'
 export const toBKTException = async (
   response: FetchResponseLike,
 ): Promise<BKTException> => {
-  const errorBody: ErrorResponse | null = await response.json()
-  const error = errorBody?.error
+  const responseText: string = await response.text()
+  const errorBody: ErrorResponse | null = (() => {
+    try {
+      return JSON.parse(responseText)
+    } catch (e) {
+      return null
+    }
+  })()
+  const { message = responseText } = errorBody?.error ?? {}
```

Comment:
Can we assume that the error response is always a string, so that I can remove this `JSON.parse()`?

Reply:
Currently, errors handled by Envoy they are returned as a string by default. But this behavior could change in the feature. So for safety, it will be better to use this try/catch just in case something changes.